src/tools/sample/sample.cpp

```diff
@@ -316,6 +321,13 @@ ClientConnectionCallback(
         printf("[conn][%p] All done\n", Connection);
         MsQuic->ConnectionClose(Connection);
         break;
+    case QUIC_CONNECTION_EVENT_RESUMPTION_TICKET_RECEIVED:
+        printf("[conn][%p] Resumption ticket received (%u bytes):\n", Connection, Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicketLength);
+        for (uint32_t i = 0; i < Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicketLength; i++) {
+            printf("%.2X", (uint8_t)Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicket[i]);
```

Comment:
Do you need the %0.2X here too?
```suggestion
            printf("%0.2X", (uint8_t)Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicket[i]);
```